simulation/sub8_gazebo/diagnostics/set_model_position_test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+from __future__ import division
+
+import rospy
+import txros
+from geometry_msgs.msg import Pose, Point, Quaternion
+from nav_msgs.msg import Odometry
+from sub8_msgs.srv import RunJob, RunJobResponse
+import tf
+from sub8_ros_tools import msg_helpers
+from nodes import job_runner
+
+from twisted.internet import reactor, defer
+import numpy as np
+
+
+@txros.util.cancellableInlineCallbacks
+def run_mission(srv, sub, nh):
```

Comment:
It's funny, this actually tests to make sure the `set_model_position` function works.
For some reason the sub can not maintain arbitrary quaternion poses in the simulator, so I had to limit to only yawing, which worked 100/100 times.